src/open_producten/utils/tests/keycloak.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2021 Dimpact
+from contextlib import contextmanager
+from unittest.mock import patch
+
+from django.apps import apps
+
+KEYCLOAK_BASE_URL = (
+    "https://keycloak.server272.ovh/realms/master/protocol/openid-connect"
```

Comment:
This is probably an instance of keycloak on one of your servers? It might be good to use the docker-compose instance in `mozilla-django-oidc-db`, that way if someone else needs to add extra cassettes at some point, the environment is reproducible for them as well (https://github.com/maykinmedia/mozilla-django-oidc-db/blob/master/docker-compose.yml)

There's a test realm that's imported with a clientid and secret `testid` / `7DB3KUAAizYCcmZufpHRVOcD0TOkNO3I` and a user with username/password `testuser` / `testuser` that you can use.

Reply:
Might be good to add it in this repo as well, similar to how OF does it https://github.com/open-formulieren/open-forms/tree/master/docker/keycloak